sfcasts/command-intro.md

```diff
@@ -88,6 +88,13 @@ in our `pressButton()` method, we would just call `execute()` on the command we
 It looks something like this:
 
 ```php
+/**
+ * @param array<string, ButtonCommandInterface> $commands
```

Comment:
don't you like better `ButtonCommandInterface[] $commands`?

Reply:
Does `ButtonCommandInterface[] === array<string|int, ButtonCommandInterface>`?

I wanted to explicitly show that it's keyed by a string (as it's important to the pressButton logic) but if you think this is too complex, I'm fine with `ButtonCommandInterface[]`. I'll also adjust the script to explain this.

Reply:
IMO, it's more complex than just `ButtonCommandInterface[]` but you're right about it being keyed by a string. Let's go your way

Reply:
Ok I added a little more of a description so hopefully it could be a learning experience for someone unfamiliar with this syntax